hrrecruitment/tests/phpunit/HRRecruitmentTestTrait.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * A trait with helper methods to be reused among this extension's tests
+ */
+trait HRRecruitmentTestTrait {
```

Comment:
In this case, you can just remove the phpunitPopulateDB() method. It doesn't seem to be used anywhere.